index.html

```diff
@@ -615,6 +622,10 @@ <h2>Creating a navigation timing entry</h2>
         <li>Set |document|'s <span>navigation timing entry</span> to |navigationTimingEntry|.
         <li>Set |navigationTimingEntry|'s <a data-for="PerformanceNavigationTiming">`Critical-CH` restart time</a>
         to |criticalCHRestart|.
+        <li>If |document| is the outermost main frame's document, set |navigationTimingEntry|'s
```

Comment:
You can say:

> If _document_'s [=node navigable=] is a [=top-level traversable=]

Reply:
Ack. Removed this part because on the HTML spec side, we set iframes' NRR to null.